fastchat/protocol/openai_api_protocol.py

```diff
@@ -100,6 +100,15 @@ class ChatCompletionStreamResponse(BaseModel):
     model: str
     choices: List[ChatCompletionResponseStreamChoice]
 
+class TokenCheckRequest(BaseModel):
+    model: str
+    prompt: str
+    max_tokens: int
```

Comment:
I think it is okay to use max_tokens for OpenAI-compatibility. Although it is not a good name, it is used by OpenAI.

Reply:
Yeah I agree, better to keep it for compliancy and thus ease of implementation for now, but hopefully the right person sees this and is in the room when making a proper API standard and we can get that sorted.